src/promptflow-devkit/promptflow/_sdk/_orchestrator/run_submitter.py

```diff
@@ -87,7 +87,7 @@ def _run_bulk(self, run: Run, stream=False, **kwargs):
             from promptflow.tracing._start_trace import start_trace
 
             logger.debug("Starting trace for flow run...")
-            start_trace(session=kwargs.get("session", None), attributes=attributes, run=run.name)
+            start_trace(collection=kwargs.get("session", None), attributes=attributes, run=run.name)
```

Comment:
why kwargs.get("session")? show session changed?

Reply:
offline synced with Brynn, we might not need this parameter here any more; let me rethink here carefully.